internal/pkg/cli/env_init.go

```diff
@@ -767,6 +782,28 @@ func (o *initEnvOpts) validateCredentials() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateInternalALBSubnets() error {
+	if len(o.internalALBSubnets) == 0 {
+		return nil
+	}
+	isImported := make(map[string]bool)
+	for _, placementSubnet := range o.internalALBSubnets {
+		for _, subnet := range append(o.importVPC.PrivateSubnetIDs, o.importVPC.PublicSubnetIDs...) {
```

Comment:
note that this might mutate the underlying array of `o.importVPC.PrivateSubnetIDs`. 
IMO it should be either [cloned](https://github.com/golang/go/wiki/SliceTricks#copy), or have two loops